Keas.Mvc/Controllers/Api/KeysController.cs

```diff
@@ -123,7 +121,7 @@ select new
         }
 
         [HttpPost]
-        [ProducesResponseType(typeof(Key), StatusCodes.Status200OK)]
+        [ProducesResponseType(typeof(KeyInfo), StatusCodes.Status200OK)]
         public async Task<IActionResult> Create([FromBody]CreateKeyViewModel model)
```

Comment:
@wongband This just returns 

            var key = new Key()
            {
                Name = model.Name.Trim(),
                Code = model.Code.Trim(),
                Notes = model.Notes,
                Tags = model.Tags,
            };
Plus the team.
I'm not sure it is worthwhile to create a partial for this... Maybe just switch back to Key?

Reply:
@wongband I think this is the way to go, but Create (Probably the other http posts too) doesn't return a counts that you have in the KeyPartial. But lets wait for @srkirkland . I'd like to get his input too.

Reply:
@jSylvestre sounds good.

Reply:
@srkirkland any thoughts about this?

Reply:
Agree with @jSylvestre here, looking at the body of the method it appears to just be returning the `new Key()` object so we should be able to just use the response type of `Key` as it was

Reply:
What I don't like, but don't have a good solution for, is that the key documentation in swagger also has all the other "includes" which are not set in this case.

Reply:
If we don't want to go the DTO route just for the sake of swagger documentation, maybe we can figure out a custom attribute solution to tell SwaggerGen to filter out unwanted properties.

Reply:
@sprucely What do you think about using JsonIgnore to not show certain variables in the Swagger UI?